### DISCUSSION TITLE
Increase version code in environment variables

### DISCUSSION THREAD

**Discussion by VadimZhuk0v** [codemagic-ci-cd/codemagic-docs#1188] [Q&A] [answered]
I want to deploy my app via firebase. I sore my code's version inside environment variables. I need to increase this value on each build inside yaml file. How can I do this?

**Comment by icarusdust** [comment, marked as answer]
Hi, please check out [this documentation](https://docs.codemagic.io/knowledge-codemagic/build-versioning/).

**Reply by VadimZhuk0v** [reply]
Hi
I've seen this page. It talks only about codemagic build number. I need my own custom variable. To be able to control the value by myself.
